tsl/src/continuous_aggs/create.c

```diff
@@ -784,6 +784,13 @@ cagg_validate_query(Query *query)
 		Dimension *part_dimension = NULL;
 		ht = ts_hypertable_cache_get_cache_and_entry(rte->relid, CACHE_FLAG_NONE, &hcache);
 
+		if (hypertable_is_distributed(ht))
+			ereport(ERROR,
+					(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
+					 errmsg("hypertable is a distributed hypertable"),
```

Comment:
I suggest: "continuous aggregates are not supported on distributed hypertables" to give more context.